community/php7/APKBUILD

```diff
@@ -75,6 +75,7 @@ makedepends="
 	net-snmp-dev
 	openldap-dev
 	pcre-dev
```

Comment:
are you sure first version still needed?

Reply:
I haven't checked all the dependencies yet, but will try to minimize them